README.md

```diff
@@ -4,7 +4,11 @@ Public Transport of Vienna/Austria Module for MagicMirror<sup>2</sup>
 
 ## Example
 
-![](.github/example.jpg) ![](.github/example2.jpg) ![](.github/example3.PNG)
+![](.github/example.jpg) ![](.github/example2.jpg) ![](.github/example3.PNG) ![](.github/example4.png)
+
+
+Matrix view
+![](.github/example-matrix.png)
```

Comment:
Added images are to big.

I'd also reduce the amount of images. Maby swap example-matrix with example2. Should be enoth.